services/server/server.go

```diff
@@ -522,6 +522,13 @@ func (s *Server) validateChangeSettingsRequest(ctx context.Context, req *serverp
 
 	// check request parameters compatibility with environment variables
 
+	if req.DisableUpdates && s.envSettings.EnableUpdates {
+		return status.Error(codes.FailedPrecondition, "Updates are enabled via ENABLE_UPDATES environment variable.")
+	}
+	if req.DisableUpdates && s.envSettings.EnableTelemetry {
+		return status.Error(codes.FailedPrecondition, "Updates cannot be disabled because telemetry is enabled via ENABLE_TELEMETRY environment variable.")
+	}
```

Comment:
updates and telemetry is independent, so we can remove this check.

Reply:
Here https://perconacorp.slack.com/archives/C03J8FZFU/p1626470324102700 is discussion that they are coupled. Ok I will change everything related to it.

Reply:
Please discuss it with Roma, Steve or Denys. But previously users were able to disable updates using env variable without disabling telemetry.

Reply:
that was just assumption from Steve (that disable doesn't work because of telemetry). It is not a requirement.

Reply:
@denisok Ok thanks for verifying.